imageroot/bin/module-dump-state

```diff
@@ -33,3 +33,11 @@ while read -r share ; do
     ls -s "${share}.plocate"
 done < <(net conf listshares)
 EOF
+
+#
+# TimescaleDB backup, see
+# https://docs.timescale.com/self-hosted/latest/backup-and-restore/logical-backup/#back-up-and-restore-an-entire-database
+#
+echo "Dumping samba_audit DB from TimescaleDB to disk:"
+podman exec -u postgres timescaledb pg_dump -p 15432 -d samba_audit -Fc -Z 5 > samba_audit.pgdump
+du -h samba_audit.pgdump
```

Comment:
Again, what's the point of this? AFAIK no other module does something like this.